augur/refine.py

```diff
@@ -313,7 +313,10 @@ def run(args):
         pass
     elif args.divergence_units=='mutations':
         if not args.timetree:
-            tt.infer_ancestral_sequences()
+            # infer ancestral sequence for the purpose of counting mutations
+            # sample mutations from the root profile, otherwise use most likely state.
+            # Reconstruct tip states to avoid mutations to N or W etc be counted
+            tt.infer_ancestral_sequences(marginal=True, reconstruct_tip_states='True', sample_from_profile='root')
```

Comment:
```suggestion
            tt.infer_ancestral_sequences(marginal=True, reconstruct_tip_states=True, sample_from_profile='root')
```

`'True'` as a string works because the value is truthy in Python, but [TreeTime expects a boolean](https://github.com/neherlab/treetime/blob/531f776bcad3b8b11fd470a403a9c06c78b07e36/treetime/treeanc.py#L497-L499).

mypy/pyright can catch these kinds of type issues, but it would only be effective if the TreeTime function has [PEP 484 style type annotations](https://peps.python.org/pep-0484/#type-aliases).

Reply:
thanks, Victor!